test/abstractarray.jl

```diff
@@ -775,3 +775,14 @@ end
 #to_shape
 @test Base.to_shape(()) === ()
 @test Base.to_shape(1) === 1
+
+#vec for multiple arrays
```

Comment:
Not at all merge-blocking, but this could be a `@testset`... I have a one track mind 😛